### PR TITLE
Security Fashion Refactor 

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
Standard/Alt outfits
Note: These are not before and after pictures, the right pictures have no coats to show the rolled up sprites.
image of cheridan hitting the CLOSE PR button

:cl:
tweak: Security uniforms have been made more professional, and more tactical.
/:cl:

Red is proven to increase aggression in mammals.

Testimonials

Sprites by Ausops, they're REAL NICE.

### CODE REVIEW

**Comment by xxalpha** [comment]
this shit again

Are you removing the beret?

**Comment by Supermichael777** [comment]
I can guarantee you will get the same reaction from the same group who whine about every pr.

**Comment by xxalpha** [comment]
I can barely distinguish the Warden from the HoS.

**Comment by Supermichael777** [comment]
yeah the warden needs more diferentation

**Comment by TrustyGun** [comment]
give the hos his greatcoat back, it made him look special and different and more powerful than the rest of security

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -12,11 +12,21 @@
...
+	armor = list(melee = 25, bullet = 15, laser = 25, energy = 10, bomb = 25, bio = 0, rad = 0)
+	dog_fashion = /datum/dog_fashion/back
+
+/obj/item/clothing/suit/armor/vest/alt
+	name = "armor"
+	desc = "An armored vest that protects against most types of damage."
 	icon_state = "armor"
 	item_state = "armor"
 	blood_overlay_type = "armor"
 	armor = list(melee = 25, bullet = 15, laser = 25, energy = 10, bomb = 25, bio = 0, rad = 0)
```
Comment (ChangelingRain) [inline code/modules/clothing/suits/armor.dm]: Remove this, it'll inherit.

**Comment by Bawhoppen** [comment]
:-1: I don't get why you feel it needs to be changed
just add these outfits as another alternative, the default stuff is perfectly fine you dickhole

**Comment by Iamgoofball** [comment]
hi im bawhoppen dont change any sprites pls

**Comment by Bawhoppen** [comment]
I don't want another security sprite war, literally just add them as an altneraitve outfit
also
>goof
>calling me stubborn

**Comment by Supermichael777** [comment]
Actually can we not add more sec alts it results in too much armor

**Comment by Bawhoppen** [comment]
then :-1:x:100:

**Comment by Zilenan91** [comment]
I like the new armor vest sprites :+1:

**Comment by KorPhaeron** [comment]
Guys the first set of sprites is the is the standard, second set is alt, the great coat still exists

**Comment by AnturK** [comment]
Here we go again.

**Comment by KorPhaeron** [comment]
The entire game being sprited by Ausops is part of my design vision

**Comment by TechnoAlchemisto** [comment]
@Supermichael777 and @xxalpha 

The left right are NOT before and after pictures. The right pictures are just with armored vests to show what the shirts look like rolled up.

**Comment by xxalpha** [comment]
Doesn't the HoS already look like that? I could swear it does.

**Comment by TechnoAlchemisto** [comment]
@xxalpha the tie makes it look hard to differentiate because of the 1px difference between the coat's sides, you can notice the differences in the pants though.

**Comment by Lumipharon** [comment]
Looks neat.

**Comment by PKPenguin321** [comment]
sec sprites war 2 electric boogaloo when

**Comment by Aloraydrel** [comment]
Sprite war now

**Comment by Bawhoppen** [comment]
oh I thought you had made berets default headwear
:+1: then

**Comment by Bawhoppen** [comment]
also as long as everything still exists

**Comment by TechnoAlchemisto** [comment]
This pr does not remove anything, the old sec sprites still exist in the code, unlike the previous incarnations which would completely remove them.

**Comment by Bawhoppen** [comment]
also should probably give the HOP and barman the old armor ones then and leave sec with the new shiny ones

**Comment by TechnoAlchemisto** [comment]
I did!

**Comment by Bawhoppen** [comment]
must have missed that in the bitter rage

**Comment by Zilenan91** [comment]
Ew no can we just replace all armor sprites with the new one because the new ones here look so much better

**Comment by KorPhaeron** [comment]
The most important question is whether or not the beret or cap should be standard

**Comment by Shapsy** [comment]
no it fucking isn't the answer is beret

**Comment by Lumipharon** [comment]
cap is for buttbandits.
Also what the hell do helmets look like with this?

**Comment by Bawhoppen** [comment]
the standard needs to stay helmet
beret is optional if you want to be a badass, cap is if you want to be a cool-man

**Comment by Bawhoppen** [comment]
also yes it needs to be I didn't notice you changed it to be when I first commented

**Comment by PKPenguin321** [comment]
backwards cap master race

**Comment by KorPhaeron** [comment]
The cap is for standard officers so the warden/hos can show off their better hats obviously

**Comment by Jordie0608** [comment]
Out of interest, are people objecting to this because they explicitly dislike these sprites or just because they prefer the current? Both? Would it be better if either style could be chosen?

**Comment by TechnoAlchemisto** [comment]
Nobody has objected to this after they have been rebuffed, as you can see by the OP's lack of thumbs down, and its plethora of hoorays.

**Comment by jughu** [comment]
I like these one 's betfer tbh

**Comment by xxalpha** [comment]
I object to this out of spite now.

**Comment by Bawhoppen** [comment]
I object until you change the default headwear back to helmet
beret does look cool but the beret should be the optional thing, helmet needs to stay default

**Comment by xxalpha** [comment]
I agree.

**Comment by PKPenguin321** [comment]
Baw is actually correct here

**Comment by TechnoAlchemisto** [comment]
works for me

**Comment by KorPhaeron** [comment]
Hats are better though?

**Comment by Lumipharon** [comment]
yes but that's not the point. helmet is your default goonhead item.
Seeing a sec officer who has taken the time to be FASHIONABLE matters, as opposed to forcing GOODLOOKS on pleb officers.

**Comment by TechnoAlchemisto** [comment]
Hats/berets are far better, but helmet wearers are confirmed scrub officers. It's good to differentiate.

**Comment by Steelpoint** [comment]
These are the worst sprites you could have picked.

When they were first made no one really liked them, they were the compromise sprites, and they were the reason we went back to Red since no one in game liked these sprites.

**Comment by CosmicScientist** [comment]
I guess forum feedback threads for PRs touching historically (and still) controversial are bad because you'd get told no.

**Comment by Incomptinence** [comment]
Looks dull as all get out its like the hos uniform was extruded from a biogenerator.

**Comment by Steelpoint** [comment]
Whenever I made sprite changes of this magnitude I made a forum feedback thread to get everyone's attention. 

Considering the historical precedent for this PR and its a major change, I'm surprised it never got a feedback thread.

**Comment by CosmicScientist** [comment]
To get the thread acknowledged:
[Security uniforms have been ruined once again.](https://www.tgstation13.org/phpBB/viewtopic.php?f=10&t=6549)

**Pull request merged**